examples/controls/pickPosition_control.js

```diff
@@ -0,0 +1,21 @@
+var pickControl = document.createElement('div');
```

Comment:
For the filename: either choose snake_case or CamelCase. (the rest of the project use CamelCase if I'm not mistaken).